### PR TITLE
Feature/issue 2

### PR DESCRIPTION
**Pull request by torimcd**
Github Issue: #2 

### Description

Initial code to create and populate AWS DynamoDB tables for Hydrocron.

### Overview of work done

Create Hyrocron_DB class with supporting methods to create tables and load SWOT hydrology shapefiles into dynamodb. 

### Overview of verification done

Added sample data shapefiles for testing and created unit tests for all methods. Tests pass with local installation of dynamoDB running.

### Overview of integration done

Full workflow of creating table, loading shapefile data, running a test query, deleting an item, and deleting the table was run with local offline dynamodb installation.

## PR checklist:

* [ ] Linted
* [ ] Updated unit tests
* [ ] Updated changelog
* [ ] Integration testing

_See [Pull Request Review Checklist](../CONTRIBUTING.md#reviewing) for pointers on reviewing this pull request_

### CODE REVIEW

**Comment by torimcd** [comment]
@frankinspace this PR is ready for review. Could you take a look when you have some time? I don't think I have access to assign you as a reviewer.

File: hydrocron_database/tests/__init__.py
Comment (frankinspace) [inline hydrocron_database/tests/__init__.py:1]: Can you move tests directory to top-level of the project? Typically we don't include the tests with the built package.

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+[tool.poetry]
+name = "hydrocron-db"
+version = "0.1.0"
+description = "Backend database configuration for Hydrocron API"
+authors = ["Your Name <you@example.com>"]
```
Comment (frankinspace) [inline pyproject.toml]:
```suggestion
authors = ["PO.DAAC <podaac@jpl.nasa.gov>"]
```

File: hydrocron_database/__init__.py
Comment (frankinspace) [inline hydrocron_database/__init__.py:1]: Rename this directory to `hydrocron_db` so it matches the package in pyproject.toml

File: hydrocron_database/tests/test_hydrocron_database.py [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+        dynamo_instance.delete_table()
+
+    assert not dynamo_instance.table_exists(TEST_TABLE_NAME)
+
+
+@pytest.fixture(scope='session', name="dynamo_instance")
+def dynamo_instance_fixture():
+    '''
+    Set up a boto3 resource connection to a local dynamodb instance. 
+    Assumes Local DynamoDB instance installed and running.
```
Comment (frankinspace) [inline hydrocron_database/tests/test_hydrocron_database.py]: I asked Vanesa about this too; can we use https://pypi.org/project/pytest-dynamodb/ to move the dynamodb local instance start/stop into the test setup?

Comment (torimcd) [reply hydrocron_database/tests/test_hydrocron_database.py]: done

File: hydrocron_database/__init__.py
Comment (torimcd) [reply hydrocron_database/__init__.py:1]: done

File: hydrocron_database/tests/__init__.py
Comment (torimcd) [reply hydrocron_database/tests/__init__.py:1]: done

**Comment by torimcd** [comment]
@frankinspace all requested changes are integrated, please review again.

**Review by frankinspace — APPROVED** [review]

**Pull request merged**